charts/e2g/Chart.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v2
+name: e2g
+description: "e2guardian Chart"
+type: application
+version: 0.0.1
+appVersion: 5.3.4 
+home: "https://github.com/e2guardian/e2guardian/"
+keywords:
+  - proxy
+  - forwardproxy 
+  - e2guardian
+  - e2g
+dependencies:
+ - name: manifests
+   version: ">=0.4.0"
+   repository: "https://bedag.github.io/helm-charts/"
+maintainers:
+  - name: NOC
+    email: noc@bedag.ch
+sources:
+  - https://github.com/e2guardian/e2guardian/
+  - https://github.com/bedag/docker-e2g
+annotations:
+  artifacthub.io/containsSecurityUpdates: "true"
+  artifacthub.io/changes: |
+    - inital release
```

Comment:
Please According to https://github.com/bedag/helm-charts/blob/master/CONTRIBUTING.md#changelog